pkg/controller/plan/kubevirt.go

```diff
@@ -1835,6 +1832,103 @@ func (r *KubeVirt) EnsurePersistentVolume(vmRef ref.Ref, persistentVolumes []cor
 	return
 }
 
+func (r *KubeVirt) CreatePvcForNfs(pvcName string) (err error) {
+	//TODO: set ownerReferenceso the PV and PVC deleted once the plan is done.
```

Comment:
the owner should be the conversion pod? anything else uses it? the migration/plan also might be used.

Reply:
still need to figure that one

Reply:
@ahadas @bkhizgiy It failed because cross namespaces. If the namespace we target is the one that have the plan - everything is fine. It's not our case.
Conversion pod is also problematic - it raises the question whether we wish the PV+PVC to be per VM migrated or reusable by the whole plan. If we are going on per VM approach we will need to update the owners *after* the conversion pod is created.

Reply:
@liranr23 umm it make more scene to use it for the entire plan I think but in that case the PVC is deleted before the conversion pod created so we need to see if we can somehow overcome this issue, maybe another solution should be check the destination namespace before running a plan and if the PVC+PV are not existing create it for the first run and use it for all future plans, then we can set its reference to the provider (and leave it there) , once we deleting the provider it will remove all these PVCs and PVs from the system, since we are using NFS it dosent really take up space from the storage, it currently set to 1 Gib but we can set it to less.

Reply:
i agree it's better to have the plan as owner. but the main problem is that we need the plan(/provider) in the target namespace which most likely will be different.